agents/sbd/fence_sbd.py

```diff
@@ -15,6 +15,8 @@
 PATH_NOT_EXISTS = -1
 PATH_NOT_BLOCK = -2
 
+SYSCONFIG_SBD = "/etc/sysconfig/sbd"
```

Comment:
It'd be better to pick up the build-time detection on the "initconfdir" in configure.ac for the sysconfig path so that it'd be  friendly as well for other distributions.

Reply:
Changed